src/StackAdmin/Azs.Fabric.Admin/custom/Get-AzsDrive.ps1

```diff
@@ -0,0 +1,187 @@
+    [System.String]
+    # OData filter parameter.
+    ${Filter},
+
+    [Parameter(ParameterSetName='List')]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Query')]
+    [System.String]
+    # OData skip parameter.
+    ${Skip},
+
+    [Parameter(ParameterSetName='List')]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Query')]
+    [System.String]
+    # OData top parameter.
+    ${Top},
+
+    [Parameter()]
+    [Alias('AzureRMContext', 'AzureCredential')]
+    [ValidateNotNull()]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Azure')]
+    [System.Management.Automation.PSObject]
+    # The credentials, account, tenant, and subscription used for communication with Azure.
+    ${DefaultProfile},
+
+    [Parameter(DontShow)]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Wait for .NET debugger to attach
+    ${Break},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Runtime.SendAsyncStep[]]
+    # SendAsync Pipeline Steps to be appended to the front of the pipeline
+    ${HttpPipelineAppend},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Runtime.SendAsyncStep[]]
+    # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+    ${HttpPipelinePrepend},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Returns true when the command succeeds
+    ${PassThru},
+
+    [Parameter(DontShow)]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [System.Uri]
+    # The URI for the proxy server to use
+    ${Proxy},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [System.Management.Automation.PSCredential]
+    # Credentials for a proxy server to use for the remote call
+    ${ProxyCredential},
+
+    [Parameter(DontShow)]
+    [Microsoft.Azure.PowerShell.Cmdlets.FabricAdmin.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Use the default credentials for the proxy
+    ${ProxyUseDefaultCredentials}
+)
+
+process {
+    # Generated cmdlet does not support {prefix}/{name} for ScaleUnit name, so extract the {name} part here
+    if ($PSBoundParameters.ContainsKey(('ScaleUnit')))
+    {
+        if ($null -ne $ScaleUnit -and $ScaleUnit.Contains('/'))
+        {
+            $PSBoundParameters['ScaleUnit'] = $ScaleUnit.Split("/")[-1]
+        }
+    }
+
+    # Generated cmdlet does not support {prefix}/{name} for StorageSubSystem name, so extract the {name} part here
```

Comment:
This pattern is needed for many cmdlets. We need to see how we can optimize this repetitive customization. @rakku-ms , @viananth .. not blocking this PR

Reply:
Looking forward to your solution, @daozha and I can only find this way.

Reply:
> This pattern is needed for many cmdlets. We need to see how we can optimize this repetitive customization. @rakku-ms , @viananth .. not blocking this PR

Ideally the generated SDK should support the nested resource so that PS don't need to take care of this.